Improve accessibility for Select (#4613)
kiwicom/orbit: packages/orbit-components/src/Select/README.md

- The `error` prop overwrites the `help` prop, due to higher priority.

> - When you have limited space of `Select`, you can use `customValueText` property where you can pass text alternative of
>   the current value. For instance, when label of selected option has `Czech Republic (+420)`, you can pass only `+420`
>   into this property and the original label will be visually hidden.

Reviewer: ```suggestion
- When you have limited space for the `Select`, you can use `customValueText` property to pass an alternative text for the current value. For instance, when the label of the selected option is `Czech Republic (+420)`, you can pass only `+420` to this property and the original label will be visually hidden.
```

Just some English 💅
